modules/oe_theme_content_entity_contact/oe_theme_content_entity_contact.install

```diff
@@ -20,18 +19,10 @@ function oe_theme_content_entity_contact_install() {
   }
 
   // Override entity view displays.
-  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_entity_contact') . '/config/overrides');
-  $displays = [
-    'core.entity_view_display.oe_contact.oe_general.default',
-    'core.entity_view_display.oe_contact.oe_press.default',
-  ];
-
-  foreach ($displays as $display) {
-    $values = $storage->read($display);
-    $config = EntityViewDisplay::load($values['id']);
-    foreach ($values as $key => $value) {
-      $config->set($key, $value);
-    }
-    $config->save();
-  }
+  $file_storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_entity_contact') . '/config/overrides');
+  $config_manager_helper = \Drupal::service('oe_content.config_manager_helper');
+  $config_manager_helper->updateConfig('core.entity_view_display.oe_contact.oe_general.default', $file_storage);
+  $config_manager_helper->updateConfig('core.entity_view_display.oe_contact.oe_general.oe_details', $file_storage);
+  $config_manager_helper->updateConfig('core.entity_view_display.oe_contact.oe_press.default', $file_storage);
+  $config_manager_helper->updateConfig('core.entity_view_display.oe_contact.oe_press.oe_details', $file_storage);
```

Comment:
Let's remove this for now and still use the old method, so when we find the right way of doing this we can change it everywhere (see https://github.com/openeuropa/oe_content/pull/228#discussion_r469233505 for more info).